### DISCUSSION TITLE
Intercept and modify request

### DISCUSSION THREAD

**Discussion by sidahmed-malaoui** [dstotijn/hetty#40] [General] [answered]
Are you willing to add breakpoints to hetty ? I really can't see myself working with a proxy without being able to intercept and modify the requests.

Thank you for your effort guys.

**Comment by dstotijn** [comment, marked as answer]
Hi @sidahmed-malaoui, yes it's on the [roadmap](https://github.com/dstotijn/hetty/projects/1). If you want you can subscribe to https://github.com/dstotijn/hetty/issues/23 to follow progress when it's being worked on.

**Reply by sidahmed-malaoui** [reply]
Thanks a lot dude, good luck with this awesome project. I really hope it will replace Burp.
